client/client-multi/client-c/src/kaa/kaa_context.h

```diff
@@ -101,7 +101,7 @@ typedef struct kaa_context_s {
     kaa_channel_manager_t       *channel_manager;        /**< See @link kaa_channel_manager.h @endlink. */
     kaa_profile_manager_t       *profile_manager;        /**< See @link kaa_profile.h @endlink. */
     kaa_user_manager_t          *user_manager;           /**< See @link kaa_user.h @endlink. */
-    kaa_event_manager_t         *event_manager;          /**< See @link kaa_event.h @endlink. */
+    void                        *event_manager;          /**< See @link kaa_event.h @endlink. */
```

Comment:
Why it becomes void*?  kaa_event.h defines opaque type "kaa_event_manager_t" which is perfect for our purposes ATM